Main.java

```diff
@@ -9,7 +10,15 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // create a scanner for user input
+    Scanner input = new Scanner(System.in);
+// Ask for the temperature
+System.out.println("Please enter a temperature you would like to convert in degrees Celsius.:");
```

Comment:
Good work. The reason I believe you are failing the test cases is because you have a colon (:) at the end of this output. Otherwise, the comments and names are good.